pkg/handler/handler.go

```diff
@@ -62,17 +65,25 @@ func (*healthCheckHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // RunWebhookServer starts the validating webhook resources for blueprint kanister resources
 func RunWebhookServer(c *rest.Config) error {
+	log.SetLogger(utils.GetLogger())
 	mgr, err := manager.New(c, manager.Options{})
 	if err != nil {
 		return errors.Wrapf(err, "Failed to create new webhook manager")
```

Comment:
```suggestion
		return errors.Wrap(err, "Failed to create new webhook manager")
```
No format specifiers used

Reply:
Done